client/src/components/Header.js

```diff
@@ -10,30 +9,36 @@ function Header() {
         <h1 className="u-text--center">
           <a href="/">Murial's Restaurant</a>
         </h1>
-
-        <section>
-          <Link to="/add">AddFood | </Link>
-          <Link to="/details">Details </Link>
-        </section>
+        <div>
+          <button
+            variant="outline"
+            style={{
+              borderRadius: "90px",
+              padding: "30px",
```

Comment:
A neat trick with centering text in a circle is to use flexbox instead of padding.  That way you can do `justify-content: center` and `align-items: center` to get perfect centering.

Reply:
Very cool CSS trick